lib/Basic/CmdLineOptions.cpp

```diff
@@ -117,6 +117,12 @@ llvm::cl::opt<bool>
     UseAssignmentValidatingSolver("debug-assignment-validating-solver",
                                   llvm::cl::init(false));
 
+llvm::cl::opt<bool>
+ExecuteFloatAsInt("execute-float-as-int",
```

Comment:
@Himeshi Perhaps the option name here better be other than `execute-float-as-int` because the implementation already substitutes float operations with int operations. How about `scaling` instead?